internal/conformance/runconformance.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+cd "$(dirname $0)"
```

Comment:
I think we need double quotes around `$0`, don't we?

Reply:
done